Dockerfile

```diff
@@ -35,7 +35,10 @@ RUN  set -x \
   && rm -rf /var/lib/apt/lists/* \
   && echo "alias ll=\"ls -lahF --color\"" >> ~/.bashrc
 
-ADD . /var/app
+ADD ./package.json /var/app/package.json
+ADD ./package-lock.json /var/app/package-lock.json
```

Comment:
There is no need to add this file since it will be overwritten by using `npm install`. However, you could keep it and use `npm this` for installing dependencies.

Reply:
IMO, you should revert changes in this file